### ISSUE TITLE
Importing of checklists

### ISSUE THREAD

**Issue by buzzykiwi** [stefankueng/EvImSync#18]
Hi
I just tried the importer and it is doing well for most formatting etc.
However, I am a big user of checklists in EverNote, and these no not re-appear as checklists in OneNote after using the importer.

I see in the .enex file that checklists and the status of individual items looks like this:

    <content>
      <![CDATA[<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!DOCTYPE en-note SYSTEM "http://xml.evernote.com/pub/enml2.dtd"><en-note><ul style="--en-todo:true;"><li style="--en-checked:true;"><div>Cable</div></li><li style="--en-checked:false;"><div>Cat biscuits</div></li><li style="--en-checked:true;"><div>Mex mix</div></li><li style="--en-checked:true;"><div>Beans</div></li><li style="--en-checked:false;"><div>Ice cream</div></li><li style="--en-checked:true;"><div>Shampoo</div></li><li style="--en-checked:true;"><div>Milk</div></li><li style="--en-checked:true;"><div>Corn chips</div></li></ul></en-note>      ]]>
    </content>

Is there any way to convert these to OneNote checklists?

**Comment by dmfiel** [comment]
Yes, checklists are a big part of my notes, so this is very important.

**Comment by Agonista-receptora-GABA** [comment]
Hi @buzzykiwi! Also a fan of checklists in Evernote here. I've just imported my notes using the tool, and all the notes with checkboxes were transformed to a usual bullet point list.
It's quite easy to do it manually (just select all items, click the bullet list once, then Ctrl+1 - at least in the older version of OneNote for Windows; BTW I can see that the OneNote for Windows 10 has problems with the transformed notes), so it should be possible to do it in the parser (especially that there are the CSS Properties `--en-todo` and `--en-checked` in the chunks what you shown). I'd even like to contribute to this codebase, but I'm not a C# developer, so it would be too hard and time-consuming for me to implement the solution. Maybe in free time? Anyway, @stefankueng could you at least confirm that it's doable? 😅 

Regards!
